packages/ChangesReloaded-Core.package/CRBrowser.class/instance/initializeMethodHistories.st

```diff
@@ -0,0 +1,8 @@
+initializing
+initializeMethodHistories
+
+	| changeRecords |
+	changeRecords := Cursor read
+				showWhile: [(self retreiveChangeRecordsFromChangesFile
+						reject: [:each | self isInImage: each]) reversed].
```

Comment:
Wait, are we reading the change file twice? This code is very similar to the one in `initializeHistoryList`